socnet/templates/mfa/totp/activate_form.html

```
@@ -15,18 +15,18 @@ <h3 class="mb-0 text-center">{% translate "Two-factor authentication" %}</h3>
   </div>
   <div class="form-floating mt-3">
     <input
-      type="text"
+      class="form-control"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)